LibraryManagement.Data/DataTransferObjects/Library_DTO.cs

```diff
@@ -1,9 +1,12 @@
 ﻿namespace LibraryManagement.Data.DataTransferObjects
 {
-    public class Genre_DTO
+    public class Library_DTO
```

Comment:
Lets make sure to follow 'Single Responsibility Pattern' here - The 'Genre_DTO' will be responsible for representing the exact DTO for a record from the Genre table.

We will want to have a separate class for Author_DTO to represent the 'Author' table, and we can have an 'BookAuthor_DTO' to represent the fields from Book and Author table together, whenever we join them.